CHANGELOG.md

```diff
@@ -4,8 +4,16 @@
 
 ### 💡 Enhancements 💡
```

Comment:
nit: extra line?

Reply:
not sure since this is how it's done, agreed the empty line looks weird but maybe necessary for github to parse? Anyway don't want to rock the boat if that worked before.